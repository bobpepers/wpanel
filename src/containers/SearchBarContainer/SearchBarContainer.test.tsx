import { shallow } from 'enzyme';
import * as React from 'react';
import { SearchBarContainer, SearchBarWrapperProps } from './';

const defaultProps: SearchBarWrapperProps = {
    selectedItems: [],
    handleSearchRequest: jest.fn(),
    handleClearSearchRequest: jest.fn(),
};

describe('SearchBarContainer test', () => {
    it('should render', () => {
        const wrapper = shallow(<SearchBarContainer {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
