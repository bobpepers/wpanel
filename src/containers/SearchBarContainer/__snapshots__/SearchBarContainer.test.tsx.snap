// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBarContainer test should render 1`] = `
<Fragment>
  <WithStyles(SearchBarComponent)
    activeItem={
      Object {
        "checked": true,
        "label": "",
        "value": "",
      }
    }
    endDate={null}
    handleAddItem={[Function]}
    handleChangeSearchValue={[Function]}
    handleChangeSelect={[Function]}
    handleClear={[Function]}
    handleEndDateChange={[Function]}
    handleEnterPress={[Function]}
    handleSendRequest={[Function]}
    handleStartDateChange={[Function]}
    index={0}
    searchValue=""
    selectedValues={
      Array [
        Object {
          "checked": true,
          "label": "",
          "value": "",
        },
      ]
    }
    startDate={null}
  />
</Fragment>
`;
